src/js/boardView.js

```diff
@@ -0,0 +1,70 @@
+let boardSize;
+const pCellClsName = "innerCell";
+
+/* eslint-disable no-trailing-spaces */
```

Comment:
If you're going to disable coding standard checks please add comment explaining why